src/StickPortal.js

```diff
@@ -92,20 +92,21 @@ class StickPortal extends Component<StickPortalPropsT, StateT> {
   }
 
   render() {
-    const { children, style, component, ...rest } = this.props
-    const Comp = component || 'div'
+    const { children, component, ...rest } = this.props
+    const Comp = component || Fragment
```

Comment:
I'm a bit uncomfortable with this change. We call `findDOMNode` on the <StickPortal /> element in _Stick.js_ and I'm not sure what this will return if there is no single wrapper DOM element. Can you maybe explain how this works? Not sure, but we might have some code in signavio/ui that relies on styling the anchor wrapper element, which makes this change seem even more dangerous.

Also, fragments have only been added in React 16.2, our peer dep is set to ^16.0. We would have to update it.

Reply:
I've changed that because the extra wrapping `div` actually screws with correctly measuring how wide the anchor node is (it would always be 100%). 

Since `StickPortal` is a class component this also doesn't mess with `findDOMNode` in `Stick` itself. I didn't test with multiple anchor nodes though. 

Styling the anchor/ anchor wrapper would probably be better done in userland when I'm not totally wrong.

Reply:
If you have some time tomorrw, we can also walk though this together to spot everything I've still forgotten :)

Reply:
OK, we can discuss tomorrow. 

If you wrap the anchor node (the entire `<Stick />` element) in a `div` with `display: inline-block`, the measuring will work correctly. This is how it's done currently.

If we want to change this, one open question would be: How can we prevent the user to pass a fragment? In this case, the findDOMNode would not work.